### ISSUE TITLE
Make explicit at Headers section that ternaries are allowed on .h files, since multiline macros are prohibited on headers and `norminette` coherently allows it since used with the `-R CheckDefine` flag

### ISSUE THREAD

**Issue by HCastanha** [42School/norminette#162]
https://github.com/42School/norminette/blob/224032de0603efa66318c1c0d32c73ab0203b871/pdf/fr.norme.tex#L292

https://github.com/42School/norminette/blob/224032de0603efa66318c1c0d32c73ab0203b871/pdf/fr.norme.tex#L320

Ces deux lignes sont contradictoires,
principalement lorsqu'il est destiné aux exercices C08 pour créer un `#define ABS(Value)` dans le header.
L'exception à la règle doit être explicite dans le document dans la section Header lorsque les ternaires sont permis dans les headers si `norminette` est appelé avec la flag `-R CheckDefine`.

---

https://github.com/42School/norminette/blob/224032de0603efa66318c1c0d32c73ab0203b871/pdf/en.norm.tex#L308

https://github.com/42School/norminette/blob/224032de0603efa66318c1c0d32c73ab0203b871/pdf/en.norm.tex#L336

These two lines are contradictory,
mainly when purposed on C08 exercises to make a `#define ABS(Value)` in the header.
Exception to rule should be make explicit in the document at the Header section since dealing with ternaries on headers is allowed as long as `norminette` is called with the `-R CheckDefine` flag.

---

https://github.com/42School/norminette/blob/224032de0603efa66318c1c0d32c73ab0203b871/pdf/pt_br.norm.tex#L312

https://github.com/42School/norminette/blob/224032de0603efa66318c1c0d32c73ab0203b871/pdf/pt_br.norm.tex#L340

As duas linhas acima se contradizem, principalmente quando há exercícios como os da lista C08 que pedem um `#define ABS(Value)` no header. A norma atualmente aprova o uso de ternários no header, desde que a `norminette` seja utilizada com a flag `-R CheckDefine`, logo o documento precisa ser atualizado para deixar explícita essa exceção à regra na seção Cabeçalhos (Headers).

**Comment by N0ich** [comment]
Hello,

I'm having a hard time understanding what you expect here. Can you rephrase your ticket?

**Comment by Flynnstones** [comment]
norminette -R CheckDefine
Error: TERNARY_FBIDDEN      (line:  15, col:  32):      Ternaries are forbidden

Faut croire que les ternaires sont interdites, hors dans le cadre de l'exercise C08 #define ABS(Value) je ne vois pas comment produire un bout de code qui n'utilise pas l'opérateur ternaire '?'.
